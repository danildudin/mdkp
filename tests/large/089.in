50 23 0
987 929 480 50 275 756 534 340 250 528 385 307 256 327 673 423 164 841 312 804 865 401 924 117 316 799 920 566 406 211 259 670 504 191 403 157 904 639 411 211 324 821 479 769 171 291 335 242 291 771 
813 935 683 218 380 198 264 80 892 24 736 248 467 966 548 761 311 993 800 150 60 699 848 290 31 310 499 950 322 542 935 954 626 945 217 824 723 507 737 672 102 40 724 83 592 298 903 459 93 578 
118 471 763 718 940 688 148 182 819 668 267 689 86 889 713 264 648 687 50 876 493 631 604 896 195 555 643 111 217 663 190 884 235 789 840 221 292 474 70 128 596 82 420 589 512 114 962 954 428 59 
586 721 879 859 613 938 263 87 553 202 242 374 739 626 743 423 689 352 963 536 992 89 51 102 295 151 695 667 511 427 451 215 384 378 355 678 326 688 965 372 598 988 940 677 974 69 980 987 275 564 
619 993 768 22 732 268 603 726 478 718 237 850 994 422 628 465 364 134 597 950 419 222 929 376 585 324 203 79 317 485 624 99 251 897 453 97 195 368 88 302 449 261 313 777 745 402 966 289 500 703 
704 765 600 636 854 728 15 159 267 743 829 928 118 834 701 63 924 458 19 674 983 13 56 607 489 566 40 457 242 65 149 567 85 284 598 626 243 60 955 760 755 802 998 214 407 88 452 62 835 909 
832 985 974 738 593 85 129 83 851 748 287 348 238 671 776 164 575 850 293 546 905 201 49 907 758 559 768 826 451 889 527 527 79 383 593 481 207 215 500 897 176 140 317 984 744 756 428 847 51 643 
73 2 623 456 505 900 451 308 737 919 681 178 969 747 73 83 407 579 360 880 595 344 915 396 691 192 8 40 705 57 178 747 925 267 656 370 201 235 706 95 970 223 289 556 953 668 993 766 625 571 
773 450 616 260 188 907 906 503 618 730 718 560 625 507 173 467 55 170 468 194 585 744 680 387 278 512 45 831 122 412 905 501 802 15 502 730 614 639 138 372 627 77 472 402 544 100 660 542 326 612 
606 743 994 665 642 561 14 689 134 751 614 5 287 385 817 738 764 170 798 921 288 442 712 579 72 922 82 833 488 794 295 526 769 702 614 40 819 932 72 590 650 207 682 370 616 126 127 565 575 52 
104 720 83 118 905 515 990 971 757 643 829 106 959 371 556 942 920 759 518 352 676 283 134 439 65 795 14 860 616 296 885 275 413 964 748 572 499 367 576 993 173 517 865 360 538 185 746 164 947 826 
19 403 472 598 893 29 898 110 705 425 566 158 753 489 47 15 398 453 300 332 563 503 15 173 855 606 299 11 897 515 249 420 714 72 922 50 8 353 744 397 783 117 390 659 71 842 695 317 483 52 
574 157 45 540 524 885 344 498 717 913 268 137 446 543 144 329 149 759 555 352 798 360 141 816 762 386 919 944 26 797 918 550 855 776 451 197 255 388 423 629 617 639 111 123 523 966 705 735 594 101 
585 511 787 272 652 526 385 22 79 294 252 461 183 240 770 354 586 816 673 9 843 720 463 835 804 76 450 584 905 720 789 407 325 132 91 897 198 869 819 383 117 844 898 869 257 840 81 487 162 849 
758 650 154 312 185 874 741 268 556 652 45 921 11 284 302 136 355 435 172 151 586 763 848 958 604 316 591 751 169 650 268 408 602 642 201 405 879 677 81 268 899 292 236 556 103 155 528 46 342 376 
262 496 973 284 190 271 775 500 162 359 778 61 369 785 65 995 944 117 649 785 23 901 866 468 408 832 118 899 490 557 322 355 744 541 371 384 280 767 144 741 186 141 239 675 27 465 850 694 443 471 
429 740 575 427 32 185 947 525 358 836 129 942 243 325 887 667 865 396 531 854 622 328 940 971 350 54 653 780 910 751 840 468 102 285 961 613 512 307 999 69 329 407 369 250 630 840 618 585 607 996 
845 154 726 239 384 42 51 997 175 188 51 746 227 371 144 579 803 410 949 188 174 553 692 825 942 530 664 743 750 371 839 966 907 816 748 131 29 657 708 137 635 128 122 59 587 483 692 679 461 622 
535 253 847 103 923 697 379 441 902 994 474 619 241 902 29 775 999 247 715 367 185 557 30 873 113 589 768 664 865 41 986 849 598 929 359 661 764 358 350 558 309 322 793 878 723 289 396 347 14 765 
995 976 791 368 984 449 432 172 53 708 484 739 326 183 367 356 93 165 303 564 582 217 778 616 567 52 767 388 762 900 779 428 446 469 922 559 647 788 884 616 15 763 459 217 749 451 67 101 858 421 
5 339 955 922 595 947 593 407 904 477 395 649 351 629 143 815 84 240 590 91 567 571 829 229 543 938 632 996 219 294 48 773 371 647 944 162 522 883 457 525 910 316 509 215 266 288 725 454 515 915 
522 959 158 106 169 211 606 46 429 123 479 744 533 148 823 830 992 394 509 963 29 183 245 664 477 445 615 159 463 610 473 364 330 733 426 671 411 490 104 772 578 568 90 373 402 321 858 538 910 498 
434 627 985 846 974 976 249 859 476 144 463 866 23 529 112 161 546 918 506 102 849 153 723 23 704 141 532 914 662 990 26 875 201 894 250 856 142 671 916 782 632 384 757 129 339 453 827 290 909 855 
233 798 570 154 136 373 881 647 197 327 607 713 743 104 670 594 302 281 958 670 343 109 907 960 473 572 300 567 349 251 80 302 967 264 297 152 193 250 389 605 580 113 279 511 90 195 212 489 18 834 
82759 35259 85846 90867 85338 51039 35223 28259 92192 64212 30746 43713 57931 10173 82878 79614 45248 27118 63985 38877 22775 86466 25119 
