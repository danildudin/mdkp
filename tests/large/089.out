17517
30
0 1 2 4 7 10 12 13 14 15 17 18 19 20 22 25 26 28 31 34 36 37 38 41 42 43 44 45 47 49 
